src/main/java/org/opensearch/ad/settings/AnomalyDetectorSettings.java

```diff
@@ -125,6 +127,12 @@ private AnomalyDetectorSettings() {}
             Setting.Property.Dynamic
         );
 
+    /**
+     * @deprecated This setting is deprecated because we need to manage fault tolerance for
+     * multiple analysis such as AD and forecasting.
+     * Use TimeSeriesSettings.COOLDOWN_MINUTES instead.
+     */
+    @Deprecated
```

Comment:
Good comment -- add "deprecated" section to PR description so that the directional information will be available.

Reply:
added